frontend/globals/viewer.ts

```diff
@@ -287,6 +267,22 @@ export class Viewer {
 
 	}
 
+	public onUnityProgress = (progress) => {
+		if (progress === 1) {
+			this.emit(Viewer.EVENT.VIEWER_INIT_SUCCESS, progress);
+		} else {
+			this.emit(Viewer.EVENT.VIEWER_INIT_PROGRESS, progress);
+		}
+	}
+
+	public onModelProgress = (progress) => {
+		if (progress === 1) {
+			this.emit(Viewer.EVENT.MODEL_LOADED, progress);
```

Comment:
I don't think this should be emitted here. We should stick to when unity actually sends you the model loaded signal.

Reply:
fixed